src/templates/about-page.js

```diff
@@ -1,23 +1,30 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { graphql } from 'gatsby'
-import { Button } from 'antd'
+import { Button, Row, Col } from 'antd'
 import Layout from '../components/Layout'
+import PreviewCompatibleImage from '../components/PreviewCompatibleImage'
+import Navbar from '../components/Navbar';
+
 
 export const AboutPageTemplate = ({
   title,
   image,
-  heading,
+  headingAboutTechConnect,
+  headingAboutMAF,
   mission,
   quote,
-  description,
+  aboutTechConnect,
+  aboutMAF,
+  image2,
+  image3
 }) => {
 
   return (
     <section className="section section--gradient">
       <div className="container">
         <div className="columns">
-          <div className="column is-10 is-offset-1">
+          <div className="column is-12 is-offset-2">
```

Comment:
Something is causing the page to have a big blank whitespace at the left and be cut off on the right; I think it might be this line... Make sure you test your local changes at varying screen sizes before committing. :-)

Here's what I see:
![Screenshot 2019-03-18 at 8 10 51 PM](https://user-images.githubusercontent.com/42818904/54577689-06e61f00-49ba-11e9-9ef8-3e094aecd870.png)